packages/host/src/Footer.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect } from 'react';
```

Comment:
Daddy! can I have a story? Make it a really boring one!

Reply:
I am hesitant to say as I am unsure of the value, but also a test to just make sure that there is text equal to the currently logged in user and store I *think* might one day help someone